ClosestCity/ClosestCity.cpp

```diff
@@ -36,7 +36,7 @@ std::string getTheClosestCityThatIsBiggerThanMe(std::vector<City> vec,
             });
 
         auto distanceToTheClosestCity = 100000;
```

Comment:
Let's change this number.
afak, we have to initialize `distanceToTheClosestCity` with some **big** number.
That's cool.
let's use numeric limits instead:
https://ideone.com/3sU7Qf
`auto distanceToTheClosestCity = std::numeric_limits<std::size_t>::max();`
With this everybody knows that the variable is the greatest number there is.

Reply:
Ok, I will handle it with next commit